algorithms/integration/integrator.py

```diff
@@ -96,9 +96,9 @@ def generate_phil_scope():
                   "number of blocks may be set to 1. If force is True then the"
                   "block size is always calculated."
 
-        max_memory_usage = 0.75
+        max_memory_usage = 0.90
```

Comment:
Configuration parameter perhaps useful - If I am happy to have chrome swapped out I could push this to e.g. 150%

Reply:
It is really inconvenient to map that desired behaviour into this number though.

Say you have 16 GB of RAM, 2 GB of swap, other stuff takes up 10 GB and chrome another 4 GB, leaving 2 GB physical RAM free. Some OS bits take up 0.5 GB swap.

So to allow to use 6 GB you may think you'd set this to 300%, but it's percent of available (2 GB physical + 1.5GB swap = 3.5GB), so 171%.

At least I'd find that mental arithmetic inconvenient, YMMV :wink:

Reply:
Yeh fair enough, however set(reasons for not making a parameter here) = ? Other than having to push a parameter through multiple onion layers?

Reply:
Vo reason. Just thinking an absolute "use x GB of memory for integration" with default `>=0.9 * available` could be more accessible than a relative value of something potentially obscure

Reply:
<- has missed something then

we have an alternative option to use?

Reply:
Right now there isn't one, but we could make one happen.